README.md

```diff
@@ -0,0 +1,112 @@
+# Microservice Utils
+
+[![Build Status](https://travis-ci.org/Cimpress-MCP/microservice-utils.js.svg?branch=master)](https://travis-ci.org/Cimpress-MCP/microservice-utils.js)
+[![npm version](https://badge.fury.io/js/microservice-utils.svg)](https://www.npmjs.com/package/microservice-utils)
+
+Utilities supporting authorization, request logging and other often required parts of microservices.
+
+## Using this library
+
+Install the library.
+
+```bash
+npm install microservice-utils
+```
+
+### Request Logger
+Logger which takes care of truncating Bearer tokens and safe JSON stringification. While by default it logs to console,
+it supports a custom logging function (e.g. to SumoLogic).
```

Comment:
I suggest to avoid referencing custom / commercial applications. Let's not have a bias in our libraries for one tool over the other.